Etabs_Adapter/Etabs_Adapter.csproj

```diff
@@ -228,6 +228,9 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
     <AssemblyName>ETABS18_Adapter</AssemblyName>
   </PropertyGroup>
+  <PropertyGroup>
+    <RunPostBuildEvent>Always</RunPostBuildEvent>
+  </PropertyGroup>
```

Comment:
Added in error? Should not require any post build events.

Reply:
Hi @peterjamesnugent ,
post build events calls above removed in latest commit 49033425304048adbfb3b9700a4f43c9bd11576c